tests/foundry/test/Loops.t.sol

```diff
@@ -0,0 +1,129 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.13;
+
+import "forge-std/Test.sol";
+
+contract LoopsTest is Test {
+    function setUp() public {}
+
+    function sumToN(uint256 n) internal pure returns (uint256) {
+        uint256 result = 0;
+        for (uint256 i = 0; i <= n; i++) {
+            result += i;
+        }
+        return result;
+    }
+
```

Comment:
These are good looking tests. Can you rebase them on master cleanly so we can see if it's working on CI?